repositories/src/main/scala/cromwell/pipeline/datastorage/dto/TypedValue.scala

```diff
@@ -0,0 +1,48 @@
+package cromwell.pipeline.datastorage.dto
+
+import java.nio.file.{ Path, Paths }
+
+import cromwell.pipeline.utils.json.AdtJsonFormatter._
+import play.api.libs.json._
+
+sealed trait TypedValue
+final case class StringTyped(value: Option[String]) extends TypedValue
+final case class FileTyped(value: Option[Path]) extends TypedValue
+final case class IntTyped(value: Option[Int]) extends TypedValue
+final case class FloatTyped(value: Option[Float]) extends TypedValue
+final case class BooleanTyped(value: Option[Boolean]) extends TypedValue
+
+object TypedValue {
+  type TypeValuePair = (String, Option[String])
+
+  implicit val typedValueFormat: OFormat[TypedValue] = {
+    implicit val pathFormat: OFormat[Path] = new OFormat[Path] {
+      override def writes(o: Path): JsObject = Json.obj("path" -> o.toString)
+
+      override def reads(json: JsValue): JsResult[Path] = JsSuccess(Paths.get((json \ "path").as[String]))
+    }
```

Comment:
1) `pathFormat` must be defined in a more common place, since it is also should be used in `ProjectFileConfigurationFormat`
2) @OstapStrashevskii already [implemented ](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/72/files) it, you can copy it and put it in the same place to reduce potential problems when someone will merge his PR.

Reply:
Thx, but if I would put it in the same place, I will have to import ProjectFile into TypedValue